mungegithub/github/github.go

```diff
@@ -1150,3 +1150,45 @@ func (config *Config) ForEachIssueDo(fn MungeFunction) error {
 	}
 	return nil
 }
+
+// ListAllIssues grabs all issues matching the options, so you don't have to
+// worry about paging. Enforces some constraints, like min/max PR number and
+// having a valid user.
+func (config *Config) ListAllIssues(listOpts *github.IssueListByRepoOptions) ([]*github.Issue, error) {
```

Comment:
Agree that's a major flaw.  I've had https://github.com/eparis/contrib/commit/4c21157894dd082132923154906d52230dd6e46f#diff-6ff804e5140f204c3e7068a3ba138ae5R1134 sitting around and thought it had been merged long ago.